### ISSUE TITLE
Possible new contribution?

### ISSUE THREAD

**Issue by yassineAlouini** [pmeier/pmeier#5]
Hello @pmeier. :wave: 

Hope you are doing well. 

It looks like dIoU PR is almost finished (except the refactoring that will be started later I guess). 

In the meantime, is there something I can work on? Thanks for your guidance and time.

**Comment by pmeier** [comment]
Hey Yassine :wave:

Other than watching the issue tracker and see if anything pops up for there is one more task that I haven't gotten to. It is more of a house keeping task, but it would help me greatly.

There are quite a few open and stale PRs regarding datasets. I've [assigned them to me](https://github.com/pytorch/vision/pulls/assigned/pmeier) but I haven't gotten around to resolve them. Unless there is major user demand (which there isn't or otherwise the PR wouldn't be stale), we will not add any new dataset to the old API. This means these PRs can be closed. Still, we could add them to the list of new datasets in pytorch/vision#3562.

If you want to help me out

- visit all PRs that I linked above
- check if they are adding a new dataset or are just related to datasets
- if they do, leave a comment that we will not add them to the old API and ping me there so I can actually close it. Please compile a list of all PRs that meet this fate and add the datasets to the list of new datasets also linked above.
- if the PR does not add a new dataset but is only related to datasets, check if the PR is still relevant. This might be harder to do so feel free to ping me here or on the PR for guidance. 

This is more maintainer work than contributor task, but I'm afraid I can't give you anything else at the moment. No worries if you don't want to take that up.

**Comment by yassineAlouini** [comment]
That works for me, thanks for taking the time and answering my request. Maintainer work is necessary work as well (even though it isn't always glamorous and isn't given much credit). :ok_hand: 

I will start working on this next week on my dedicated day.

**Comment by yassineAlouini** [comment]
@pmeier I have added one first comment here: https://github.com/pytorch/vision/pull/2910#issuecomment-1114659738

Let me know if this is what you were expecting from me. :ok_hand:

**Comment by pmeier** [comment]
Yes, that is what I wanted :slightly_smiling_face: Two comments though:

1. One thing I often struggle myself, is to be more "friendly" to contributors. Put yourself in their shoes: they submitted a PR with quite some work and it never got properly reviewed for whatever reason. Now we are telling them that we are going to close if they don't put in more work. That can be quite discouraging what we are trying to avoid. Try to explain the situation more to hopefully keep them as contributors.
2. Please drop the the "we" meaning "`torchvision` maintainers including you" for now. Although I tasked you do to this, it makes it sound like you are already a maintainer for the contributor. You can phrase it like a question targeted at me, like 

> In the meantime, should this PR be closed @pmeier ?

**Comment by pmeier** [comment]
Please also apply this to the comments you have made on other PRs.

**Comment by yassineAlouini** [comment]
Thanks for the suggestions @pmeier, I will update the comment accordingly.

**Comment by yassineAlouini** [comment]
A better comment (hopefully :smile:) here: https://github.com/pytorch/vision/pull/2690#issuecomment-1114678355

Let me know if this is more what you had in mind @pmeier.

**Comment by pmeier** [comment]
Yes, that is a lot better. Thank you!

**Comment by yassineAlouini** [comment]
A list of new datasets (WIP) not yet referenced [here](https://github.com/pytorch/vision/issues/3562) (should I add a comment there later?):


- C++ Caltech Dataset: https://github.com/pytorch/vision/pull/1121

**Comment by pmeier** [comment]
I don't think we will have C++ datasets unless there is significant user demand, which doesn't seem to be the case.

In any case, I'm surprised that everything else is already on the list. Well, this means Aditya did quite a good job keeping the list up to date. 

Keep up the good work @yassineAlouini!

**Comment by yassineAlouini** [comment]
Yes, it seems that there is already a good part of the work done. I like this triage task since it allows me to explore further the code base and the state of development, so thanks for the assignment. :ok_hand:

**Comment by oke-aditya** [comment]
> Aditya did quite a good job keeping the list up to date.


My part time hobby / addiction is to triage stuff and add todo to what is pending :) 

I try to keep lists as updated as possible. But feel free to ping me if I miss any. Also feel free to search stuff out in wild like latest papers on arxiv / repos over GitHub and comment if something is interesting or interests you :)

@yassineAlouini

**Comment by yassineAlouini** [comment]
@oke-aditya I have few candidates in mind but I wanted to help with existing work (since the number of contributors is finite and the work is almost infinite :stuck_out_tongue: ). 

Joke aside, I will check if something needs help once I finish triaging dataset related issues (almost done) and if there is nothing, I will propose something. :ok_hand:

**Comment by pmeier** [comment]
@oke-aditya If that is a pet peeve of yours, I might have another task for you :innocent: I'll be on parental leave soon and I'm currently swamped with other things. We have a project board for the transform rework: https://github.com/pytorch/vision/projects/5. It should include all PRs and issues that are related to that, but it seems no one besides me used it yet. I'll make sure that they will in the future, but that still leaves everything in between to be done.

Could you look through all the [issues](https://github.com/pytorch/vision/issues?q=is%3Aissue+created%3A%3E%3D2022-04-19) and [PRs](https://github.com/pytorch/vision/pulls?q=is%3Apr+created%3A%3E%3D2022-04-19) on and after the 19th of April (the last time the board was updated) and either

- add it directly to the board. Not sure if you have permissions to do that though. It can be done through the "Projects" section in the right column below labels and the like.
- add it to a list and post it here so I can do that. Please do not comment on the PRs / issues to avoid additional noise.

**Comment by oke-aditya** [comment]
> I'll be on parental leave soon and I'm currently swamped with other things. 

Congratulations Philip. My best wishes.

I cannot access the project board. But I will post the list here :smiley:

**Comment by pmeier** [comment]
> I cannot access the project board. 

Wait, does that mean you cannot see it or you can't edit it?

**Comment by oke-aditya** [comment]
Can see but not edit.

**Comment by yassineAlouini** [comment]
Congratulations @pmeier for the happy event. :tada:
